src/main/java/lol/ui/WinnerScreen.java

```diff
@@ -0,0 +1,38 @@
+package lol.ui;
+import javax.swing.*;
+import java.awt.*;
+
+public class WinnerScreen {
```

Comment:
This file is using Java Swing which is another Java graphical component.

It is not a big deal, for if you want me to merge your code, you should use Java FX instead.

(You will still get bonus points if you don't change it).